plan.md

```diff
@@ -10,19 +10,51 @@ Note: to edit on HackMD https://hackmd.io/6fFy3_y6SOWonOYeGaNiHw?edit
 
 ## Content
 Why and when? Example applications.
+- Mention the iris example, pitch it as historical, but also as a
+  botanical and agriculture problem.
+  The benefit of this example is that it forces to think about
+  measurement, but also because it has one class that is easy to separate
+  only one feature
+    - Aurelie: real irises for the video?
```

Comment:
Look at Andy's book there is a picture there with an appropriate license that we can reuse.

Reply:
Right, I was thinking to have actual real irises on the video. We need to see if it's easy to do.